.expeditor/verify.pipeline.yml

```diff
@@ -47,32 +47,42 @@ steps:
     executor:
       docker:
         image: ruby:2.6
-        privileged: true
         environment:
           - BUNDLE_GEMFILE=/workdir/Gemfile
           - FORCE_FFI_YAJL="ext"
 
-# Allow jruby to softfail since it was configured so in travis too
-- label: run-lint-and-specs-jruby
+- label: ":ruby: 3.0"
   command:
     - ruby --version
     - bundle --version
     - gem update --system
-    - apt-get update -y && apt-get install -y git make gcc
     - bundle install --without development_extras --jobs 3 --retry 3 --path vendor/bundle
-    - gem install ffi json psych
+    - gem install yajl-ruby json psych
     - bundle exec rake
-  soft_fail:
-      - exit_status: 1
   expeditor:
     executor:
       docker:
-        image: jruby
+        image: ruby:3.0
+        environment:
+          - BUNDLE_GEMFILE=/workdir/Gemfile
+          - FORCE_FFI_YAJL="ext"
+
+- label: ":ruby: 3.1"
+  command:
+    - ruby --version
+    - bundle --version
+    - gem update --system
+    - bundle install --without development_extras --jobs 3 --retry 3 --path vendor/bundle
+    - gem install yajl-ruby json psych
+    - bundle exec rake
+  expeditor:
+    executor:
+      docker:
+        image: ruby:3.1
         privileged: true
         environment:
           - BUNDLE_GEMFILE=/workdir/Gemfile
-          - RUBY_PLATFORM="jruby"
-          - FORCE_FFI_YAJL="ffi"
+          - FORCE_FFI_YAJL="ext"
 
 - label: "run specs :windows:"
```

Comment:
Looks like we would need to add the new windows distro that runs ruby 3.1 for testing here?

Reply:
Added the ruby-3.1 test for windows using windows distro docker image.